app.py

```diff
@@ -733,6 +733,18 @@ def prepare_model_args(request_body):
         if message:
             messages.append({"role": message["role"], "content": message["content"]})
 
+    authenticated_user_details = get_authenticated_user_details(request_headers)
```

Comment:
should this be conditionally enabled? or only if the developer has opted in?

Reply:
Added MS_DEFENDER_ENABLED env var to enable this as first step